### PR TITLE
Deploy flutter to gce

### LINKED ISSUE #2805: Deploy a Flutter app to Google Compute Engine
### Proposed title of article
Deploy a Flutter app to Google Compute Engine

### Introduction paragraph (2-3 paragraphs):
Google provide various hosting options including Compute Engine, App Engine, Kubernetes, Firebase, Cloud Run, and Cloud functions. The option that one goes for depends on the nature of the app/site they wish to deploy. In this tutorial, we will see to deploy a flutter application to Google Compute Engine. 
In the end we will have created a Google Cloud project, installed a Virtual Machine, created a Flutter web app and deployed it on our tech stack hosted on the Virtual Machine.
### Key takeaways: 
1. Creating a flutter web app.
2. Creating Google Cloud Project on Google Cloud Console.
3.  Creating Virtual Machine instance on Google Compute Engine
4. Deploying the created app on the GCE.

### References: 
Please list links to any published content/research that you intend to use to support/guide this article.

### Templates to use as guides
- [How To Guide Template](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/how-to-guide-template.md)
- [Software Review Template](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/software-review-template.md)
- [Tutorial Template](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/tutorial-template.md)

### PR DESCRIPTION
**Pull request by paulromans**
### Article checklist

> If you have not already, please go over our [Resources Page](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/resources-page.md) for more tips and tricks on how to improve your overall technical writing so reviews are swifter, increase the chance of a payout and provide more value to our readers.

> For first-time contributors and for more details on our submission guidelines, **see our [Contributing Guidelines](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/CONTRIBUTING.md)**.

#### Formatting and structure checks
- [x] Have you used the correct folder and file structure? - **see [Contribution Guidelines](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/CONTRIBUTING.md)**
- [x] Is your article properly formatted in Markdown? - **see [Github Markdown guide](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)**
- [x] Have you used our preferred heading structure? - **[H3 (###) and up](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#headers)**
- [x] Does your article follow any of the suggested structure formats? - **see [example formats](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md#example-format-structure)**

- [x] Is your article over 750 words? - **Tool**: [Wordcounter.net](https://wordcounter.net)
- [x] Is the article you are submitting an in-depth and unique article? Does it go beyond what is in the official docs and what is covered in other blog sites. See these articles as [examples](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/resources-page.md).
- [x] Does your article provide enough value and detail about your topic? - **Articles should be clear, accurate, and fully explained.**
- [x] Can your article be understood by beginners? - **Assume the audience is smart but has no prior exposure to the common terminology in your article.**

- [x] Have you included a hero image that is 600x400 pixels and have the copyright to use it? - **Tools:** [Pixlr Image Editor](https://pixlr.com/e) for resizing and [Unsplash](https://unsplash.com) for Creative Commons images
      
#### Grammar & plagiarism checks
- [x] Have you spellchecked and grammar checked your article? - **Tools:** [Write&Improve](https://writeandimprove.com/?lang=en-GB) - [ProWritingAid](https://prowritingaid.com/) - [Grammarly](https://grammarly.com) 
- [x] Please place your article through a 3rd party plagiarism checker. We suggest using Quetext, [this tool](https://www.quetext.com/) is free and has a daily limit. **We typically accept articles with 10% or less.**
- [x] Have you checked your article for readability? - **Tool:** [Hemmingway](http://www.hemingwayapp.com)
- [x] Have you added sources for quotes and images that aren't yours?

#### Technical checks
- [x] Are your code snippets properly formatted for syntax highlighting - **see [Syntax guide](https://gohugo.io/content-management/syntax-highlighting)**
- [x] Have you checked your code runs correctly and you've highlighted all necessary dependencies for installation?
- [x] Are the software programs and packages you're highlighting in your article up to date, using current versions and not deprecated?

This closes #2805

> Finally, delete the article checklist notes in blockquotes and submit your PR. We look forward to reviewing your article.

### CODE REVIEW

**Comment by WanjaMIKE** [comment]
Plagiarism check done - Proceed.

**Comment by mercymeave** [comment]
Hello @paulromans, Thank you for writing such a clear piece, kindly add a hero image for your article to finalize.

**Review by mercymeave — APPROVED** [review]
@paulromans 

Congratulations on such an easy to read article.

Looking forward to getting more articles from you.

Lets wait for a final review on this one.

File: content/articles/deploy-flutter-to-computer-engine/index.md
Comment (WanjaMIKE) [inline content/articles/deploy-flutter-to-computer-engine/index.md:65]: Add correct link

File: content/articles/deploy-flutter-to-computer-engine/index.md
Comment (WanjaMIKE) [inline content/articles/deploy-flutter-to-computer-engine/index.md:74]: Add correct link

File: content/articles/deploy-flutter-to-computer-engine/index.md
Comment (WanjaMIKE) [inline content/articles/deploy-flutter-to-computer-engine/index.md:99]: `... to clone the application`

File: content/articles/deploy-flutter-to-computer-engine/index.md
Comment (WanjaMIKE) [inline content/articles/deploy-flutter-to-computer-engine/index.md:121]: add correct link

File: content/articles/deploy-flutter-to-computer-engine/index.md
Comment (WanjaMIKE) [inline content/articles/deploy-flutter-to-computer-engine/index.md:125]: Correct link

File: content/articles/deploy-flutter-to-computer-engine/index.md
Comment (WanjaMIKE) [inline content/articles/deploy-flutter-to-computer-engine/index.md:139]: Break sentence into two

File: content/articles/deploy-flutter-to-computer-engine/index.md
Comment (WanjaMIKE) [inline content/articles/deploy-flutter-to-computer-engine/index.md:164]: link

File: content/articles/deploy-flutter-to-computer-engine/index.md
Comment (WanjaMIKE) [inline content/articles/deploy-flutter-to-computer-engine/index.md:167]: Replace `flutter` with `Flutter`

File: content/articles/deploy-flutter-to-computer-engine/index.md
Comment (WanjaMIKE) [inline content/articles/deploy-flutter-to-computer-engine/index.md:32]: Rephrase `Everything with Google requires a project accessed using an API key`

**Comment by WanjaMIKE** [comment]
Unfortunately, this article is not ready for final review. Please go through the above comments and correct all errors/issues. Thanks

**Comment by hectorkambow** [comment]
@paulromans 

Good afternoon and thank you for submitting your topic suggestion.
 
Not a problem - it seems this PR is still quite a bit aways from being completed. 
 
As a rule of thumb - please only submit articles (pull requests) with a complete and fully polished article - ready to be published. 
 
All revisions and edits should be completed on your own forked repo (as to not take up room in the queue). 
 
We will be closing this and please re-submit when it is ready to be published as a new PR (and you can add the duplicate label) and ping the reviewer and myself. 


cc @WanjaMIKE 

cc @mercymeave - please go over suggestions by the content moderators to improve potential future reviews.

**Pull request closed without merging**
